crypto/src/main/java/net/corda/v5/crypto/merkle/MerkleProof.java

```diff
@@ -20,17 +20,25 @@
 public interface MerkleProof {
 
     /**
-     * Get the type of the proof.
-     *
-     * @return Type of the proof.
+     * Returns the type of the proof.
      */
     @NotNull
     MerkleProofType getProofType();
 
+    /**
+     * Returns the size of the {@link MerkleTree} out of which this proof was created.
+     */
     int getTreeSize();
 
+    /**
+     * Returns disclosed plain data of a {@link MerkleTree} which are to be verified that
+     * they are indeed part of a {@code MerkleTree}.
+     */
     List<IndexedMerkleLeaf> getLeaves();
 
+    /**
+     * Returns the hashed data supplementing the plain data.
```

Comment:
Returns the hashed intermediate node data supplementing the plain data.

Reply:
Suggestion applied